manual/guides/courses.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: manual
+title: Course Planning & Registration
+subtitle: "Information on Choosing and Registering for Classes"
+permalink: /manual/guides/courses
+---
+
+# Undergraduate Students
+
+A list of courses for all students can be found [here](http://catalog.illinois.edu/courses-of-instruction/npre/), and for undergraduate students, the NPRE department has compiled an interactive suggested course sequence for undergraduate students, which can be found [here](https://npre.illinois.edu/academics/undergraduate/curriculum).
+Undergraduate Students should meet with their faculty advisor each semester to discuss what other courses they need to take to satisfy their degree requirements. They should bring with them a copy of their DARS report as well as a plan for the coming semester.
+
+# Graduate Students
+
+Each graduate student in ARFC should have a folder in Google Drive shared between only them and Prof. Huff. In that folder should be a Student Plan in the form of a spreadsheet.
+
+<img src="https://user-images.githubusercontent.com/393899/84724236-44cf4b80-af4d-11ea-9ac4-8ce729b72679.png" alt="example-spreadsheet" width="750"/>
+
+Before registration for the coming semester, which is around the end of April for the coming Fall semester or the beginning of November for the coming Spring semester, graduate students should fill out their course plan and discuss it with Prof. Huff. Use these links to find the course requirements for the [MS degree](http://catalog.illinois.edu/graduate/engineering/nuclear-plasma-radiological-engineering-ms/) and [PhD degree](http://catalog.illinois.edu/graduate/engineering/nuclear-plasma-radiological-engineering-phd/).
+
+Early Research Assistantship students should be taking 2-3 graduate classes per semester and 4-8 research credit hours. Later, students typically take one or fewer graduate classes per semester and fill the rest of their necessary hours with research credits.
+
+After Prof. Huff approved your course plan in the Student Record, you can request that the registration hold be lifted so that you can go ahead and register for classes. To do so, send a polite email to the student office in NPRE (specifically, the graduate student advisor staff member in NPRE), CC'ing Prof. Huff, with a pdf copy of your student record document attached. Let the staff member know that we have discussed the attached course plan, Prof. Huff has approved it, and you would like to request the registration hold to be lifted.
+
+Once this is done, you can go through [Self Service](https://apps.uillinois.edu/selfservice/) and Register for Classes by going to "Registration & Records", then "Enhanced Registration".
```

Comment:
Registration for Katy's section of Thesis research can only be done through "Classic Registration."

There are some additional steps required for adjusting the credit hours.

1. Enter the CRN for Prof. Huff's section
2. Update selection
3. Update credit hours
etc

These are the kind of specifics that would be helpful. e.g. I forget Katy's CRN every semester. I think its like 24115 but I'm not sure.

Reply:
How do you update the number of hours for the section? And what sort of certificates do you think are useful?